.prettierrc.json

```diff
@@ -1,7 +1,7 @@
 {
   "parser": "typescript",
   "singleQuote": true,
-  "printWidth": 100,
+  "printWidth": 80,
```

Comment:
please remove the prettier config for not (use only default values). see https://github.com/Thomaash/vis-configs